web/beacon-app/src/features/auth/hooks/useSubmitLogin.tsx

```diff
@@ -48,11 +50,22 @@ const useSubmitLogin = ({ setData, onReset, onSetCurrentUserEmail }: Props) => {
   }, [authenticated, navigate, auth?.access_token]);
 
   useEffect(() => {
-    if (status === 'paused') {
-      toast.error(`${ErrorMessage.NETWORK_ERROR}`);
+    if (hasUnverifiedEmailError) {
+      toast.error(
+        <div className="flex flex-col gap-5">
+          <p>
+            <Trans>Please verify your email address and try again!</Trans>
```

Comment:
I saw this error message appear twice on the log in page. One toast message has the button, but the other didn't. I also saw a toast error twice after clicking to resend an email on the verify-account page. I've included screenshots below.

<img width="1677" alt="Screenshot 2023-10-03 at 11 31 38 AM" src="https://github.com/rotationalio/ensign/assets/94616884/bc476e89-9027-4733-8b54-6c4ecbc10f71">

<img width="1680" alt="Screenshot 2023-10-03 at 11 31 18 AM" src="https://github.com/rotationalio/ensign/assets/94616884/b8757cde-3238-4a8f-afcc-35b81a961323">